packages/bot-utils/src/lib/novel/NovelV3.ts

```diff
@@ -121,6 +121,7 @@ export interface NovelMap {
   id: string;
   name: string;
   description: string;
+  isGlobal: boolean;
```

Comment:
I suggest using
```suggestion
  isGlobal?: boolean;
```
so it's backwards compatible with the previous type